runtime/development/src/lib.rs

```diff
@@ -114,6 +116,7 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 impl_opaque_keys! {
 	pub struct SessionKeys {
 		pub aura: Aura,
+		pub block_rewards: BlockRewards,
```

Comment:
Since we couldn't use SessionManager because (i dont think it supports tuples) and you need to know the aura key changes so can adjust the collator stake accordingly, I think it makes sense. 
It seems clean, although I am not able to understand how the properties inside the SessionKeys struct makes the compiler happy about the "whole" SessionKeys struct implementing successfully the SessionHandler trait.